test/base/CustomExportsBaseSpec.scala

```diff
@@ -160,18 +164,19 @@ trait CustomExportsBaseSpec
       .withCSRFToken
   }
 
-  def withNewCaching(dataToReturn: ExportsDeclaration) {
-    when(mockExportsCacheService.getItemByIdAndSession(any[String], any[String]))
+  // FIXME should be promoted to MockExportsCacheService trait
+  override def withNewCaching(dataToReturn: ExportsDeclaration) {
+    when(mockExportsCacheService.getItemByIdAndSession(any[String], ArgumentMatchers.eq(dataToReturn.sessionId)))
       .thenReturn(Future.successful(dataToReturn.items.headOption))
 
     when(
       mockExportsCacheService
-        .update(any[String], any[ExportsDeclaration])
+        .update(ArgumentMatchers.eq(dataToReturn.sessionId), any[ExportsDeclaration])
```

Comment:
Just a placeholder. As it was discussed, please change it to `any[String]` and verify inputs per test case.